src/components/Main/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Main';
```

Comment:
The code seems fine for exporting the Main component, but it would be great if you could share the other files in the pull request so that I can provide a more comprehensive review. In particular, please share the files where you have implemented the adding and deleting of todos.